recipes/nfs-client.rb

```diff
@@ -5,13 +5,7 @@
 ::Chef::Resource::RubyBlock.send(:include, MhOpsworksRecipes::RecipeHelpers)
 include_recipe 'mh-opsworks-recipes::create-metrics-dependencies'
 
-storage_info = node.fetch(
-  :storage, {
-    export_root: '/var/tmp',
-    network: '10.0.0.0/8',
-    layer_shortname: 'storage'
-  }
-)
+storage_info = get_storage_info
```

Comment:
The diffs here are a bit confusing as it seems `get_storage_info` was previously already defined but not being used here for some reason.